src/site/content/en/fast/fetch-priority/index.md

```diff
@@ -21,27 +22,31 @@ tags:
 
 When a browser parses a web page and begins to discover and download resources such as images, scripts, or CSS, it assigns them a fetch `priority` in an attempt to download resources in an optimal order. These priorities can depend on the kind of resource and where it is in the document. For example, in-viewport images may have a `High` priority while the priority for early loaded, render-blocking CSS via `<link>`s in the `<head>` could be `Very High`. Browsers are pretty good at assigning priorities that work well but may not be optimal in all cases.
 
-In this article, we'll discuss Priority Hints and the `fetchpriority` attribute, which allow you to hint at the relative priority of a resource (`high` or `low`). Priority Hints can help optimize the Core Web Vitals.
+In this article, we'll discuss the Fetch Priority API and the `fetchpriority` HTML attribute, which allow you to hint at the relative priority of a resource (`high` or `low`). Fetch Priority can help optimize the Core Web Vitals.
+
+{% Aside %}
+This feature was originally called Priority Hints but was renamed to Fetch Priority after standardization. See [history](#history) below for more details.
+{% endAside %}
```

Comment:
Also move this up below the compat widget, as this is a page-level notice about the rename.

```suggestion
{% Aside %}
This feature was originally called Priority Hints but was renamed to Fetch Priority after standardization. See [History](#history) below for more details.
{% endAside %}
```